app/next-client-app/app/datasets/[id]/page.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbLink,
+  BreadcrumbList,
+  BreadcrumbSeparator,
+} from "@/components/ui/breadcrumb";
+import { columns } from "./columns";
+import { getScanReports } from "@/api/scanreports";
+import { DataTable } from "@/components/data-table";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
+import { objToQuery } from "@/lib/client-utils";
+import { ScanReportsTableFilter } from "@/components/scanreports/ScanReportsTableFilter";
+import { FilterParameters } from "@/types/filter";
+import { getDataSet } from "@/api/datasets";
+
+interface DataSetListProps {
+  params: {
+    id: string;
+  };
+  searchParams?: { status__in: string } & FilterParameters;
+}
+
+export default async function ScanReports({
+  params: { id },
+  searchParams,
+}: DataSetListProps) {
+  const defaultParams = {
+    hidden: false,
+    page_size: 10,
+    parent_dataset: id,
+  };
+  const combinedParams = { ...defaultParams, ...searchParams };
+
+  const query = objToQuery(combinedParams);
+  const scanReports = await getScanReports(query);
+  const datasetName = await getDataSet(id);
+  const filter = <ScanReportsTableFilter filter="dataset" filterText="name" />;
+
+  return (
+    <div className="pt-10 px-16">
+      <div>
+        <Breadcrumb>
+          <BreadcrumbList>
+            <BreadcrumbItem>
+              <BreadcrumbLink href="/">Home</BreadcrumbLink>
+            </BreadcrumbItem>
+            <BreadcrumbSeparator>/</BreadcrumbSeparator>
+            <BreadcrumbItem>
+              <BreadcrumbLink href="/datasets">Datasets</BreadcrumbLink>
+            </BreadcrumbItem>
+            <BreadcrumbSeparator>/</BreadcrumbSeparator>
+            <BreadcrumbItem>
+              <BreadcrumbLink href={`/datasets/${id}/`}>
+                {datasetName.name}
+              </BreadcrumbLink>
+            </BreadcrumbItem>
+          </BreadcrumbList>
+        </Breadcrumb>
```

Comment:
suggestion: may be a separate component, where we simply pass the target URL and display name.

Reply:
this is a good idea. But it is also kind of complicated (and needs long props) to re-use the component in other, deeper pages (Scanreports Fields/Values, for example) cause it will need different id, tableids, ... and names for the links. If you have any methods to handle this, please let me know. Cheers!

Reply:
No worries. You could try creating a list containing display names and their corresponding URLs, and then passing this list as a prop. But, this may add overhead due to the need to create and manage the list.

Reply:
Absolutely - this is a bit of technical debt I've been willing to accept, as I feel we _could_ spend a day (or so) working out the perfect breadcrumb component. We'll add it to the backlog